.gitignore

```diff
@@ -21,4 +21,12 @@ docs/source/reference/generated/
 # codecov
 .coverage
 codecov
-coverage.xml
\ No newline at end of file
+coverage.xml
+
+# vcrpy
+usa_00136.dat.gz
```

Comment:
You'll eventually need to commit these into the `tests` folder so that `vcrpy` can find them during testing, so you probably don't want this in the gitignore.